lib/scarpe/stack.rb

```diff
@@ -1,5 +1,9 @@
 # frozen_string_literal: true
 
+require "net/http"
+require "openssl"
+require "nokogiri"
```

Comment:
we dont nokogiri in few other examples so ig its not an issue to add nokogiri here?

Reply:
I think it's going to be needed by the Webview display library no matter what, yeah.